### ISSUE TITLE
Cannot run tutorial scripts - Fatal Python error: PyThreadState_Get: no current thread

### ISSUE THREAD

**Issue by AntonioSerrano** [Microsoft/malmo#511]
I am a beginner tinkering with Malmo for the first time. I got through the process of installation successfully (I can launch Minecraft from terminal with launchClient.sh). But I got the following error when running the first Python tutorial script 'tutorial_1.py' on terminal:

```
Fatal Python error: PyThreadState_Get: no current thread
Abort trap: 6
```

My current OS version is macOS Sierra 10.12.4. Any clue?

**Comment by AntonioSerrano** [comment]
I have been investigating. The error is thrown when trying to import the MalmoPython lib.

**Comment by AntonioSerrano** [comment]
Solved. It seems like my default Python version does not match that of the system, and as stated in the [quick guide](https://github.com/Microsoft/malmo) malmo currently only support the system python. It is weird because my default Python version is also 2.7 but it seems like it is not exactly the same. Anyway, this is what I did:
1) First, launch the Minecraft client in the background. On terminal just type:
`./launchClient.sh &`
Do not forget the `&` symbol.
2) Back on terminal, instead of typing `python tutorial_1.py`, replace it by `/usr/bin/python tutorial_1.py`.
